src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Controllers/AccountController.cs

```diff
@@ -270,7 +272,7 @@ public async Task<IActionResult> ConfirmEmail(string userId, string code)
             var user = await this._accountService.FindByIdAsync(userId).ConfigureAwait(false);
             if (user == null)
             {
-                throw new ArgumentException($"Nie można załadować użytkownika z identyfikatorem '{userId}'.");
+                throw new ArgumentException(@ErrorDuringEmailConfiguration,  @userId);
```

Comment:
have you checked that? It will not show a proper message - you should use `String.Format`

Reply:
To be honest. As far I understand what I am doing it looks correct:
![obraz](https://user-images.githubusercontent.com/27915290/89529909-dd67a680-d7ed-11ea-93d8-f940b108dcf9.png)

Reply:
it should be `Wystąpił nieoczekiwany błąd podczas konfigurowania wiadomości e-mail dla użytkownika z identyfikatorem '1'`

Reply:
done